tests/Benchmarks/Microsoft.Experimental.Collections/MultiValueDictionaryPerformanceTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.Collections.Extensions.Tests
+{
+    public class MultiValueDictionaryPerformanceTests
+    {
+        [Params(1000, 10000, 10000)]
+        public int Size { get; set; }
+
+        public int RandomKey { get; set; }
+
+        public MultiValueDictionary<int, int> dict;
+        
+        [GlobalSetup(Targets = new[] { nameof(Add), nameof(Clear), nameof(GetItem), nameof(GetKeys), nameof(ContainsValue) })]
```

Comment:
Interesting use of GlobalSetup. I didn't know you could specify benchmark specific setups like this. Good to know :)